src/core_deku/contracts.ml

```diff
@@ -0,0 +1,40 @@
+module Conversions : Smart_contracts.Conversions = struct
+  open Core
+
+  module Address = struct
+    include Address
+
+    let size = 36
+
+    let to_bytes t = Bytes.of_string (to_string t)
+
+    let of_bytes t =
+      let addr = Address.of_string (Bytes.to_string t) in
+      Option.value_exn addr
+  end
+
+  module Ticket_id = struct
+    include Ticket_id
+
+    let size t = 36 + Bytes.length t.data
```

Comment:
I guess using `Address.size` is better than using the hardcoded value.

Reply:
changed